src/evaluation_system/api/plugin_manager.py

```diff
@@ -71,7 +71,7 @@
 PLUGIN_ENV = "EVALUATION_SYSTEM_PLUGINS"
 """Defines the environmental variable name for pointing to the plug-ins"""
 
-IMAGE_RESIZE_EXCEPTIONS = ["PDF", "HDF5", "GRIB"]
+IMAGE_RESIZE_EXCEPTIONS = ["PDF", "HDF5", "GRIB", "GIF"]
```

Comment:
Wouldn't it be better to resize the gif, instead of only copying it. Large gif's can make the page basically unloadable.

Reply:
that is happening by default with mp4 as well. The only reason mp4 were copied is because this suffix did not fall in any category and passed thru.
Although probably mp4 do not load so much in memory. I will take a look

I reverted this change, I am clossing this PR, to separate the issues.